### ISSUE TITLE
Mapping issues

### ISSUE THREAD

**Issue by Blackpaw** [revenz/FileFlows#65]
All the files under one directory failed with a mapping issue

I'm original thought it was the ':' in the directroy name, but it turned out to be the '\\' , doesn't display correctly in the logging either,

'AFK: The Video Game / Fantasy' is actually 'AFK: The Video Game\\Fantasy' ('\\' embedded in the path name)

A tricky path parsing problem and a bit of an edge case. I was quite happy to remove the embedded '\\' and FF processed it fin after a rescan and restart,

```
==============================================================================
===                      PROCESSING NODE OUTPUT START                      ===
==============================================================================
2022-05-15 17:15:53.895 - INFO -> Base URL: http://docker-server:5000
2022-05-15 17:15:53.937 - INFO -> Address: fileflows-node
2022-05-15 17:15:53.945 - INFO -> Making request[GET]: http://docker-server:5000/api/node/by-address/fileflows-node?version=0.6.0.938
2022-05-15 17:15:53.981 - INFO -> Waiting on node task
2022-05-15 17:15:54.205 - INFO -> Completed node task
2022-05-15 17:15:54.205 - INFO -> Node SignalrUrl: http://docker-server:5000/flow
2022-05-15 17:15:54.207 - INFO -> Making request[GET]: http://docker-server:5000/api/library-file/15cc89c1-6abd-45e3-8dbe-d3be769b9412
2022-05-15 17:15:54.275 - INFO -> Making request[GET]: http://docker-server:5000/api/library/a394942d-2991-441f-ab93-31a93225470a
2022-05-15 17:15:54.378 - INFO -> Making request[GET]: http://docker-server:5000/api/library-file/exists-on-server/15cc89c1-6abd-45e3-8dbe-d3be769b9412
2022-05-15 17:15:54.414 - ERRR -> Library file exists but is not accessible from node: /mnt/plex/TV/AFK: The Video Game / Fantasy Web series/Season 1/AFK The Webseries.S01E08.IRL.mkv
2022-05-15 17:15:54.437 - INFO -> Making request[PUT]: http://docker-server:5000/api/library-file
2022-05-15 17:15:54.488 - INFO -> Exit Code: 0
==============================================================================
===                       PROCESSING NODE OUTPUT END                       ===
==============================================================================


```

**Comment by revenz** [comment]
closing this as a very low priority/edge case... if others report it ill re-look at it.
